non-code.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Non-Code
+layout: default
+permalink: /non-code/
+---
+
+While the focus of this site is open source software, we recommend checking out [Creative Commons](http://creativecommons.org/choose/) for most other types of projects.
```

Comment:
What about "there are other licenses more appropriate for projects that aren't code, e.g. Creative Commons or the Open Font License"?